### ISSUE TITLE
latest version can't run due to spec mismatch for darwinia

### ISSUE THREAD

**Issue by shunjizhan** [polkadot-js/apps#6224]
https://github.com/polkadot-js/apps/blob/e8a3cae9195ce720c046377d855429522e22a015/packages/apps-config/package.json#L14

when I install `@polkadot/apps-config@latest`, dependency `@darwinia/types` version was 1.2.1, which I think has a spec mismatch for specName `pangoro`, so I was getting err
```
Uncaught TypeError: _spec.default[specName] is not iterable at getBundleFromSpecName
```

This was fixed in latest version `@darwinia/types@1.2.50`, which is included in the `@polkadot/apps-config` beta version, but I have to manually install the beta version to avoid the above bug.

**Comment by jacogr** [comment]
So it is fixed, not sure why it is logged.

Stable releases of the apps UI only goes out when it is ready. The betas always has the latest code.

**Comment by shunjizhan** [comment]
sounds good : )

just thinking for better programmer UX, latest version should at least be able to run, hopefully we can publish the beta version to @latest so other ppl won't get this error

**Comment by jacogr** [comment]
No, releases of stable/latest are done when it is ready. Same all all other libs, i.e. API publishes weekly. Before that it would mean that latest may have dependencies on other beta versions, which is a much greater nightmare.

**Comment by shunjizhan** [comment]
kk, that make sense, totally understood that stable versions should be more stable before it's publish haha, thanks!
